drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -1208,12 +1207,13 @@ RDD::TextureID RenderingDeviceDriverD3D12::texture_create(const TextureFormat &p
 
 		// If views of different families are wanted, special setup is needed for proper sharing among them.
 		// Two options here:
-		// 1. If ID3DDevice10 is present and driver reports relaxed casting is, leverage its new extended resource creation API (via D3D12MA).
-		// 2. Otherwise, fall back to an approach based on abusing aliasing, hoping for the best. [[CROSS_FAMILY_ALIASING]]
+		// 1. If the driver reports relaxed casting is, leverage its new extended resource creation API (via D3D12MA).
+		// 2. Otherwise, fall back to an approach based on having multiple versions of the resource and copying as needed. [[CROSS_FAMILY_FALLBACK]]
 		if (p_format.shareable_formats.size()) {
 			if (format_capabilities.relaxed_casting_supported) {
 				ComPtr<ID3D12Device10> device_10;
 				device->QueryInterface(device_10.GetAddressOf());
+				DEV_ASSERT(device_10);
```

Comment:
The above comment was updated to reflect the device10 casting is no longer used for checking but an assert and an unused cast remain here. Perhaps we can relax this and just not check if device10 is present considering it doesn't use it anymore?

Reply:
Makes sense!